src/components/reusables/Header.tsx

```diff
@@ -1,13 +1,16 @@
 import * as React from "react";
+import { LogoutIcon } from "@nypl/dgx-svg-icons";
 
 export interface HeaderProps {
   text?: string;
   imgSrc?: string;
   alt?: string;
+  logOut?: string;
 }
 
 export default class Header extends React.Component<HeaderProps, void> {
   render(): JSX.Element {
+    let src = this.props.imgSrc ? require(`${this.props.imgSrc}`) : "";
```

Comment:
You are not actually using this variable. I think you want to use it on line 20.